src/main/java/com/minecolonies/coremod/commands/ActionMenuType.java

```diff
@@ -0,0 +1,301 @@
+            CheckForAutoDeletesCommand.class,
+            new ActionArgument("confirmDelete", ActionArgumentType.BOOLEAN, ActionArgumentType.Is.OPTIONAL)
+            )),
+    WHO_AM_I(new ActionMenu(
+            "WhoAmI",
+            "whoami",
+            ForgePermissionNodes.WHO_AM_I,
+            WhoAmICommand.class
+            )),
+    WHERE_AM_I(new ActionMenu(
+            "WhereAmI",
+            "whereami",
+            ForgePermissionNodes.WHERE_AM_I,
+            WhereAmICommand.class
+            )),
+
+    LIST_COLONIES(new ActionMenu(
+            "ListColonies",
+            "list",
+            ForgePermissionNodes.LIST_COLONIES,
+            ListColoniesCommand.class,
+            new ActionArgument("page", ActionArgumentType.INTEGER, ActionArgumentType.Is.OPTIONAL),
+            new ActionArgument("abandonedSinceTimeInHours", ActionArgumentType.INTEGER, ActionArgumentType.Is.OPTIONAL)
+            )),
+    REQUEST_SYSTEM_RESET_ALL(new ActionMenu(
+            "RequestSystemResetAll",
+            "rsResetAll",
+            ForgePermissionNodes.REQUEST_SYSTEM_RESET_ALL,
+            RSResetAllCommand.class
+            )),
+
+    LIST_CITIZENS(new ActionMenu(
+            "ListCitizens",
+            "list",
+            ForgePermissionNodes.LIST_CITIZENS,
+            ListCitizensCommand.class,
+            new ActionArgument("colony", ActionArgumentType.COLONY, ActionArgumentType.Is.REQUIRED),
+            new ActionArgument("page", ActionArgumentType.INTEGER, ActionArgumentType.Is.OPTIONAL)
+        )),
+    KILL_CITIZEN(new ActionMenu(
+            "KillCitizen",
+            "kill",
+            ForgePermissionNodes.KILL_CITIZEN,
+            KillCitizenCommand.class,
+            new ActionArgument("colony", ActionArgumentType.COLONY, ActionArgumentType.Is.REQUIRED,
+                    new ActionArgument("citizen", ActionArgumentType.CITIZEN, ActionArgumentType.Is.REQUIRED))
+        )),
+    RESPAWN_CITIZEN(new ActionMenu(
+            "RespawnCitizen",
+            "respawn",
+            ForgePermissionNodes.RESPAWN_CITIZEN,
+            RespawnCitizenCommand.class,
+            new ActionArgument("colony", ActionArgumentType.COLONY, ActionArgumentType.Is.REQUIRED,
+                    new ActionArgument("citizen", ActionArgumentType.CITIZEN, ActionArgumentType.Is.REQUIRED))
+        )),
+    CITIZEN_INFO(new ActionMenu(
+            "Info",
+            "info",
+            ForgePermissionNodes.CITIZEN_INFO,
+            CitizenInfoCommand.class,
+            new ActionArgument("colony", ActionArgumentType.COLONY, ActionArgumentType.Is.REQUIRED,
+                    new ActionArgument("citizen", ActionArgumentType.CITIZEN, ActionArgumentType.Is.REQUIRED))
+        )),
+
+    SHOW_COLONY_INFO(new ActionMenu(
+            "ShowColonyInfo",
+            "info",
+            ForgePermissionNodes.SHOW_COLONY_INFO,
+            ShowColonyInfoCommand.class,
+            // TODO: need OR() ActionArgumentType
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1135)